docs/quickstart.md

```diff
@@ -0,0 +1,428 @@
+
+Let's make you definition valid again. If you look at the defined Field Names of the [Operation object](https://spec.openapis.org/oas/latest.html#operation-object) (after the warning description), you can quickly detect what's wrong with your definition file - there cannot be a Field Name with the `operationIdentifier` name, it is invalid. So, change it back to the `operationId` and you're done. In this way, you will fix both the warning and the error because the unexpected property `operationIdentifier` is gone.
+
+To check that everything is correct, run the `lint` command again:
+
+```bash
+$ openapi lint petstore.yaml
+No configurations were defined in extends -- using built in recommended configuration by default.
+
+validating petstore.yaml...
+petstore.yaml: validated in 47ms
+
+Woohoo! Your OpenAPI definition is valid. 🎉
+```
+
+Congratulations! You did a great job and now your OpenAPI definition is valid again!
+
+### split - divide your large definition into smaller constituent parts
+
+With this command, your OpenAPI definition file will be divided into constituent parts to follow the multi-file approach.
+
+In your terminal, type the following:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+```
+
+You should get the following response:
+
+```bash
+$ openapi split petstore.yaml --outDir openapi
+Document: petstore.yaml is successfully split
+ and all related files are saved to the directory: openapi 
+
+petstore.yaml: split processed in 44ms
+```
+
+Cool, split completed. But what actually happened?
+
+Open the `openapi` directory. See? That's where the magic took place:
+
+```bash
+.
+├── components
+│   └── schemas
+│       ├── Error.yaml
+│       ├── Pet.yaml
+│       └── Pets.yaml
+├── paths
+│   ├── pets.yaml
+│   └── pets@{petId}.yaml
+└── openapi.yaml
+```
+
+With the `split` command, you can easily switch to a multi-file structure in a blink of an eye even if your definition file became extremely lengthy.
+
+### bundle - combine several definition files into a single one
+
+With this command, you can merge standalone files back into a single definition file. `bundle` does the opposite of the [`split`](#split) command.
+
+Let's bundle the definition file that you split earlier.
+
+In your terminal, type the following:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+```
+
+You should get the following response:
+
+```bash
+$ openapi bundle openapi/openapi.yaml --output bundled.yaml
+bundling openapi/openapi.yaml...
+📦 Created a bundle for openapi/openapi.yaml at bundled.yaml 46ms.
+```
+
+You should now see a new `bundled.yaml` file as a result of the `bundle` command completion. Open this file in your text editor or IDE and compare it with the linted definition file. You can see that both files are the same which means that the `bundle` command executed flawlessly.
```

Comment:
FYI, it won't always be identical, but it will be identical in meaning. For example, if there were originally comments in the yaml file, those would be lost.

Reply:
Okay, got it.
In this case, I believe, we can simply say that "You can see that OpenAPI definition hasn't changed"